longevity_test.py

```diff
@@ -154,6 +154,13 @@ def run_prepare_write_cmd(self):
             for stress in verify_queue:
                 self.verify_stress_thread(cs_thread_pool=stress)
 
+        prepare_end_cql_cmds = self.params.get('prepare_end_cql_cmds')
+        if prepare_end_cql_cmds:
+            self._run_cql_commands(prepare_end_cql_cmds)
```

Comment:
It's nice functionality, but I think it deserve better naming.

Reply:
Suggestions for such name ?

Reply:
`post_prepare_cql_cmds`